docs/source/models/concept_models/vivarium_nutrition_optimization/pregnancies/concept_model.rst

```diff
@@ -605,6 +621,14 @@ Specific outputs for specific models are specified in the following section.
     - GBD maternal disorders parent cause is equal to the *sum* of maternal disorders sub-causes. Therefore, the incidence of the aggregate maternal disorders cause is quite high relative to the rate of pregnancies and when it is increased due to risk effects from hemoglobin, the calculated probability of an incident case can be greater than one. Since these probabilities are capped at one, we end up underestimating the incidence rate of maternal disorders at a population level. Note that this issue was present in the IV iron implementation; however, in the nutrition optimization implementation, maternal disorders mortality is conditional on maternal disorders incidence (whereas mortality was correlated with incidence, but not conditional on incidence in the IV iron implementation). Therefore, we are slightly underestimating maternal disorders mortality in this model.
     - As the underestimation is slight, we will move forward despite this limitation. In the meantime, we will investigate possible solutions to address this issue (in particular, modeling each individual maternal disorders sub-cause within the simulation), which we may consider incorporating into the model after other higher priority updates are made (such as intervention implementations in models 6 and 7)
     - TBD
+  * - Uncertainty in the intervention impact on hemoglobin applied as individual heterogeneity rather than draw-level uncertainty
+    - Effect size uncertainty represents statistical uncertainty about the estimation of the population mean difference rather than a measure of standard deviation in measured individual responses to the intervention
+    - Update how we handle uncertainty for this parameter
+    - Bugfix run
+  * - No baseline calibration of IFA effect on hemoglobin
+    - IFA effect on hemoglobin should be adjusted for baseline coverage as described in the intervention model document so that those with no baseline coverage experience a hgb change equal to -hgb_shift * baseline_coverage and those with baseline coverage experience a hgb change equal to hgb_shift * (1 - baseline_coverage)
+    - Update how we handle baseline coverage
```

Comment:
Whoops, I basically shot right past that code snippet in the docs. This baseline adjustment makes sense, but now I'm a bit confused about how the interventions "stack" with respect to the baseline scenario.

Reply:
This is looking good in the latest run, removed the V&V issue in the latest commit